packages/spanner/src/session-pool.js

```diff
@@ -45,6 +45,8 @@ var util = require('util');
  *     at any given time. (Default: `0`)
  * @param {number} options.keepAlive - How often to ping idle sessions, in
  *     minutes. Must be less than 1 hour.
+ * @param {number} options.acquireTimeout - Time before giving up trying to
```

Comment:
This should either be `acquireTimeoutMs` or just mention we expect this in milliseconds.